test/unit/collection.test.ts

```diff
@@ -6,8 +6,12 @@ import { cleanup, createServer, HELLO } from '../tools/mongodb-mock';
 
 describe('Collection', function () {
   let server = null;
-  beforeEach(() => createServer().then(_server => (server = _server)));
-  afterEach(() => cleanup());
+  beforeEach(async () => {
```

Comment:
It's a fixup but it came up because I had to set a break point and see a stack trace that came from here. I would have to make this little code change again if I were to need to debug an issue here again, might as well keep it.

Reply:
okay, fine by me